scripts/run_social_messaging.py

```diff
@@ -94,6 +94,10 @@ def run_social_sinks(msg_port, social_twitter, social_slack):
 
 
 if __name__ == '__main__':
+    # This is early in startup to ensure the PANOPTES logging code is installed early.
+    pocs_config = load_config(config_files=['pocs'])
```

Comment:
Just a note that it does `pocs` by default so you could just `load_config()` edit: and I see you are just moving it anyway.

Reply:
I'll drop the config_files=['pocs'].